docs/api/hacks/getVersion.mdx

```diff
@@ -7,7 +7,13 @@ description: ''
 
 # getVersion
 
-In Valtio, updates to proxied objects are tracked internally with a version number. Therefore `getVersion` helper can be used to check if a proxied object has been updated. This is not typically useful or recommended to use in application code.
+In Valtio, updates to proxied objects are tracked internally with a version number.
+
+Every mutation to a proxy increases a global version number, and assigns the just mutated proxy, and any parent proxies (which automatically subscribe to their child proxies), to the latest version number.
```

Comment:
fwiw, the reason we use the global number is to detect loop, that support circular structure. If we don't need to support circular structure, local version number just works.

Reply:
Ah! That's cool. I noticed it handles cycle really well.

I probably won't work that tidbit into the docs, but I might make an in-source comment for it... :-)